atlasdb-backup/src/main/java/com/palantir/atlasdb/backup/AtlasBackupService.java

```diff
@@ -178,7 +205,7 @@ private void storeBackupToken(AtlasService atlasService, InProgressBackupToken b
     /**
      * Completes backup for the given set of atlas services.
      * This will store metadata about the completed backup via the BackupPersister.
-     *
+     * <p>
```

Comment:
?

Reply:
my intellij seems to not autoformat anymore, I tried doing it via the menu and it did things to the javadocs.